xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxVobsub.h

```diff
@@ -56,10 +57,8 @@ class CDVDDemuxVobsub : public CDVDDemux
     CStream(CDVDDemuxVobsub* parent)
       : m_discard(AVDISCARD_NONE), m_parent(parent)
```

Comment:
should be `m_discard(false)`. there's also a loop in the .cpp file that always evaluates to false: `while(m_Streams[current->id]->m_discard == AVDISCARD_ALL)`